tests/unit/common/test_table.py

```diff
@@ -912,7 +912,8 @@ class SubTable(DynamicTable):
         {'name': 'col6', 'description': 'optional region', 'table': True},
         {'name': 'col7', 'description': 'required, indexed region', 'required': True, 'index': True, 'table': True},
         {'name': 'col8', 'description': 'optional, indexed region', 'index': True, 'table': True},
-        {'name': 'col10', 'description': 'required, indexed vocab column', 'index': True, 'class': VocabData},
+        {'name': 'col10', 'description': 'required, indexed enum column', 'index': True, 'class': EnumData},
+        {'name': 'col11', 'description': 'required, enumerable column', 'enum': True, 'index': True},
```

Comment:
What is the difference (if any) between these two lines?

Reply:
The first should test passing in the class directly. The second should test just doing it with the keyword.